bin/tester/runtimes.js

```diff
@@ -97,17 +99,26 @@ export default async function getRuntimeSpecs(url, { verbose = false } = {}) {
   }
 
   try {
-    const specs =
+    const stories =
       typeof dom.window.__chromaticRuntimeSpecs__ === 'function' &&
       !dom.window.__chromaticRuntimeSpecs__.isDeprecated
         ? await dom.window.__chromaticRuntimeSpecs__()
         : await extract(dom.window);
 
-    return specs;
+    return {
+      specs: stories.map(toSpec),
+      options: stories.length ? getOptions(stories[0].parameters) : {},
+    };
   } catch (err) {
     throw err;
   } finally {
     // cleanup
     dom.window.close();
   }
 }
+
+const getOptions = ({ options }) => ({
+  hierarchyRootSeparator: options.hierarchyRootSeparator,
+  hierarchySeparator: options.hierarchySeparator,
+  storySort: options.storySort ? options.storySort.toString() : undefined,
```

Comment:
Do you think we should pass this up to the server, or should we instead just sort the stories before passing them up?

Reply:
(Not psyched about `eval`-ing user input in our server node process ;)

Reply:
Yea, I wasn't sure about it either.

Reply:
Ok, let's maybe just sort client side then.